src/sas/qtgui/Utilities/GridPanel.py

```diff
@@ -180,13 +181,17 @@ def addFitResults(self, results):
         self.tabWidget.setTabToolTip(self.tabWidget.count()-1, model_name)
         self.data_dict[page_name] = results
 
-    @classmethod
-    def onHelp(cls):
+    
+    def onHelp(self):
         """
         Open a local url in the default browser
         """
         url = "/user/qtgui/Perspectives/Fitting/fitting_help.html#batch-fit-mode"
-        GuiUtils.showHelp(url)
+        try:
+            self.helpWindow = DocViewWindow(parent=self.parent, source=url)
+            self.helpWindow.show()
+        except Exception as ex:
+            logging.warning("Cannot display help. %s" % ex)
```

Comment:
Identical code appearing in multiple places. Can you do it once and share code?

Reply:
`DocViewWIndow` is the shared code and replaces the old `GuiUtils.showHelp(url)`.  We have added a try except which could perhaps move to DocViewWindow? Otherwise I'm not sure I understand the question?